test/integration/proportion-boolean/patients/patient-numer-bundle.json

```diff
@@ -0,0 +1,205 @@
+            }
+          ]
+        },
+        "status": "completed",
+        "subject": {
+          "reference": "Patient/b6fa120e-931c-48da-93e2-3e38c264ae8f"
+        },
+        "performedPeriod": {
+          "start": "2022-11-04T14:02:04.516Z",
+          "end": "2022-11-05T14:02:04.516Z"
+        }
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Procedure/6dfece5d-c508-4503-bb09-f6713c8b3065"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Procedure",
+        "id": "db689c3f-57b9-4b51-a785-119bedf6d6fd",
+        "code": {
+          "coding": [
+            {
+              "system": "exampleSystem3",
+              "version": "exampleVersion3",
+              "code": "exampleCode3"
+            }
+          ]
+        },
+        "status": "completed",
+        "subject": {
+          "reference": "Patient/b6fa120e-931c-48da-93e2-3e38c264ae8f"
+        },
+        "performedPeriod": {
+          "start": "2022-05-21T14:35:24.685Z",
+          "end": "2022-05-22T14:35:24.685Z"
+        }
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Procedure/db689c3f-57b9-4b51-a785-119bedf6d6fd"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "MeasureReport",
+        "id": "fe990e03-b2fc-48f3-a8be-481d52586f66",
+        "measure": "http://example.com/Measure/measure-ProportionBooleanTest",
+        "period": {
+          "start": "2022-01-01T05:00:00.000Z",
+          "end": "2022-12-31T05:00:00.000Z"
+        },
+        "status": "complete",
+        "type": "individual",
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/test-case-cqfm"
+          ]
+        },
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-inputParameters",
+            "valueReference": {
+              "reference": "#e2af7c69-3656-42fb-80cb-571664b275e5"
+            }
+          }
+        ],
+        "modifierExtension": [
+          {
+            "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-isTestCase",
+            "valueBoolean": true
+          }
+        ],
+        "contained": [
+          {
+            "resourceType": "Parameters",
+            "id": "e2af7c69-3656-42fb-80cb-571664b275e5",
+            "parameter": [
+              {
+                "name": "subject",
+                "valueString": "b6fa120e-931c-48da-93e2-3e38c264ae8f"
+              }
+            ]
+          }
+        ],
+        "group": [
+          {
+            "population": [
+              {
+                "code": {
+                  "coding": [
+                    {
+                      "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                      "code": "initial-population"
+                    }
+                  ]
+                },
+                "count": 0
```

Comment:
It doesn't _really_ matter for the sake of this integration test, but maybe in the future if we ever want to do anything with the test case measure report, we should properly reflect it in fqm-testify.

AKA can you actually select the desired populations in testify before exporting the patient